arwen/contexts/runtime.go

```diff
@@ -285,27 +289,24 @@ func (context *runtimeContext) SetReadOnly(readOnly bool) {
 	context.readOnly = readOnly
 }
 
-func (context *runtimeContext) SetInstanceContextID(id int) {
-	context.instanceContextDataPointers = append(context.instanceContextDataPointers, &id)
-	context.instance.SetContextData(unsafe.Pointer(&id))
-}
-
 func (context *runtimeContext) SetInstanceContext(instCtx *wasmer.InstanceContext) {
-	context.instanceContext = instCtx
+	// TODO remove this method when the vmContextMap is removed
```

Comment:
this function does not do anything. you can safely remove it.

Reply:
Removed, along with `GetInstanceContext()`.